src/mongo/db/pipeline/TDigest.h

```diff
@@ -93,14 +93,14 @@ class TDigest {
    * Returns a new TDigest constructed with values merged from the current
    * digest and the given sortedValues.
    */
-  TDigest merge(presorted_t, Range<const double*> sortedValues) const;
```

Comment:
Same as above. I would suggest we only need this merge method.
`TDigest merge(const std::vector<double>& sortedValues) const;`